eng/restore/binplacePackages/binplacePackages.depproj

```diff
@@ -33,6 +33,10 @@
       <Version>4.5.3</Version>
     </PackageReference>
 
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Condition="'$(TargetsNetStandard)' != 'true' OR '$(NETStandardVersion)' &gt;= 2.0">
```

Comment:
As discussed in the standup, please add a choose when otherwise element in the file that makes sure that nothing gets restored for netcoreapp that is unintended. And please also add comments.